### DISCUSSION TITLE
Private mode and mutli screens

### DISCUSSION THREAD

**Discussion by StrasGus** [rustdesk/rustdesk#6039] [General]
Hello guys, in firt time: congratulations for great dev job !
I send you a message because when I activate the private mode on client device, the host device show the private frame only on one screen (I have 3), anoters are free to showing...

Do you know this behavior ? Have you any fix or solution ?

Bye !

**Comment by rustdesk** [comment]
private mode has no multiple display support yet.

**Comment by StrasGus** [comment]
this feature is it sheduled ?

Cordialement,

Guillaume Ruffier.
________________________________
De : RustDesk ***@***.***>
Envoyé : dimanche 15 octobre 2023 05:10
À : rustdesk/rustdesk ***@***.***>
Cc : StrasGus ***@***.***>; Author ***@***.***>
Objet : Re: [rustdesk/rustdesk] Private mode and mutli screens (Discussion #6039)


private mode has no multiple display support yet.

**Comment by Rack1538** [comment]
Sorry, but I'm trying to find what Private Mode is supposed to do. Can you tell me?

**Reply by discoveryX64** [reply]
It blanks out the hosts screen. That's about it.

**Reply by Rack1538** [reply]
Thanks!
